wiki/installing-windows.md

```diff
@@ -10,11 +10,29 @@ Once the boot menu is open, move the cursor using the arrow keys to the name of
 
 ¹ Can be usually USB HDD or your USB drive model. Your mileage may vary.
 
-## Installing
+## Disk Conversion (if needed)
 
-::: danger
-The disk that you are trying to install Windows on must be a GPT disk if you are on a UEFI system or MBR if you are on a BIOS system, otherwise it will not let you install. Use `diskpart` to change the partition scheme. ***THIS DELETES ALL DATA ON THE DISK***
-:::
+The disk that you are trying to install Windows on must be a GPT disk if you are on a UEFI system or MBR if you are on a BIOS system, otherwise it will not let you install. 
```

Comment:
```suggestion
The disk that you are trying to install Windows on, must be a GPT disk if you are on a UEFI system, or MBR if you are on a BIOS system, otherwise you will get an error.
```